index.html

```diff
@@ -0,0 +1,122 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Survey Form</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <div class="container">
+      <h1 id="title">Our <strong>Survey</strong> Form</h1>
+
+      <!-- <section id="hero">
+        <div class="hero container">
+          <div>
+            <h1>Hello, <span></span></h1>
+            <h1>My Name is <span></span></h1>
+            <h1>Hamza <span></span></h1>
+            <a href="#" type="button" class="cta">Portfolio</a>
+          </div>
+        </div>
+      </section> -->
+
+      <p id="description">
+        You want to know more about our coffee? Send us a message.
+      </p>
+      <form id="survey">
+        <label for="survey-form">Name</label>
+        <input
+          type="text"
+          id="survey-form"
+          name="name-label"
+          placeholder="Name"
+        />
+
+
+
+
+
+        <label for="email">Email</label>
+        <input
+          type="email"
+          id="email"
+          name="email-label"
+          placeholder="Email"
+          pattern="email@example.com"
+        />
+        <label for="number-label">Number</label>
+        <input
+          type="number"
+          id="number-label"
+          name="number-label"
+          min="9"
+          max="15"
+          placeholder="Num"
+          required
+        />
+        <div CLASS="dropdown">
+          <label for="dropdown">Choose a number:</label>
+          <select name="dropdown-label" id = "dropdown">
```

Comment:
Please, use space-less equal signs

Reply:
- [ ] Great job spotting 👍🏼 this error, but as mentioned before it will be great to make your comments look more like suggestions instead of orders. And kindly add some `emoji` to make your review look more friendly 😄 ✅